Resources/Prototypes/_CM14/Entities/Clothing/pouches.yml

```diff
@@ -656,15 +656,36 @@
     #whitelist: # todo cm14 add cassettes
 
 - type: entity
-  parent: CMPouch
+  parent: Clothing
   id: CMPouchMachete
   name: H6B pattern M2132 machete scabbard
   description: A large leather scabbard used to carry a M2132 machete. It can be strapped to the pouch slot.
   components:
   - type: Sprite
     sprite: _CM14/Objects/Clothing/Pouches/machete.rsi
-  - type: Storage
-    maxItemSize: Large
-    whitelist:
-      tags:
-      - Machete
+    state: sheath
+  - type: Item
+    size: Huge
+  - type: Clothing
+    quickEquip: false
+    slots:
+    - pocket
+  - type: ItemSlots
+    slots:
+      item:
+        name: Machete
+        insertVerbText: sheath-insert-verb
+        ejectVerbText: sheath-eject-verb
+        whitelist:
+          tags:
+          - Machete
+  - type: ItemMapper
+    mapLayers:
+      sheath-sabre:
```

Comment:
Can you try renaming these to sheath-machete (including the png) and see if it still works? I don't see anything in the item mapper code that would make it not work...